code/modules/food_and_drinks/kitchen_machinery/kitchen_machine.dm

```diff
@@ -290,6 +238,15 @@
 			return
 		make_recipes(recipes_to_make)
 
+/obj/machinery/kitchen_machine/proc/dispose()
```

Comment:
could you make this proc take `mob/user` as a parameter instead of using `usr` ?

Reply:
dispose() comes from ui_act, which doesnt have user either. Should I add a mob/user here anyways and have it called from that usr in dispose?

Reply:
ui_act can actually access a `mob/user` the `datum/tgui/ui` parameter for the proc has a `user` var